generative_ai/batch_predict/gemini_batch_predict_gcs.py

```diff
@@ -13,38 +13,35 @@
 # limitations under the License.
 import os
 
-from vertexai.preview.batch_prediction import BatchPredictionJob
-
 PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
 
+# TODO(developer): Add your Cloud Storage bucket details here
+INPUT_BUCKET = "cloud-samples-data"
+OUTPUT_BUCKET = "python-docs-samples-tests"
 
-def batch_predict_gemini_createjob(
-    input_uri: str, output_uri: str
-) -> BatchPredictionJob:
-    """Perform batch text prediction using a Gemini AI model.
-    Args:
-        input_uri (str): URI of the input file in Google Cloud Storage.
-            Example: "gs://[BUCKET]/[DATASET].jsonl"
 
-        output_uri (str): URI of the output folder in Google Cloud Storage.
-            Example: "gs://[BUCKET]/[OUTPUT].jsonl"
-    Returns:
-        batch_prediction_job: The batch prediction job object containing details of the job.
-    """
+def batch_predict_gemini_createjob() -> str:
+    "Perform batch text prediction using a Gemini AI model and returns the output location"
 
     # [START generativeaionvertexai_batch_predict_gemini_createjob]
     import time
     import vertexai
 
     from vertexai.preview.batch_prediction import BatchPredictionJob
 
-    # TODO(developer): Update and un-comment below lines
-    # input_uri ="gs://[BUCKET]/[OUTPUT].jsonl" # Example
-    # output_uri ="gs://[BUCKET]"
-
     # Initialize vertexai
     vertexai.init(project=PROJECT_ID, location="us-central1")
 
+    input_uri = f"gs://{INPUT_BUCKET}/batch/prompt_for_batch_gemini_predict.jsonl"
+    # Sample filedata
+    #     {"id": 1, "request": {"contents": [{"parts": {"text": "Give me a recipe for banana bread."}, "role": "user"}]}}
+    #     {"id": 2, "request": {"contents": [{"parts": {"text": "Give me a recipe for banana bread."}, "role": "user"}]}}
+
+    output_uri = f"gs://{OUTPUT_BUCKET}/batch/batch_text_predict_output"
+    # Sample filedata for expected output file
+    #     {"candidates":[{"avgLogprobs":"NaN","citationMetadata":{"citations":[{"endIndex":310,"startIndex":179,"uri":"https://myhoustonmajic.com/tag/dessert/"},{"endIndex":414,"startIndex":284,"uri":"http://forums.soompi.com/en/topic/215719-stained-glass-window-cookies/"},{"endIndex":1045,"startIndex":919,"u...
```

Comment:
Thanks the format looks but the content looks weird.. !?

1. The urls are pointing to http://forums.soompi.com/en/topic/215719-stained-glass-window-cookies/ & https://myhoustonmajic.com/tag/dessert/ (there is n't anything worth looking)

2. Above response does not seem to match the Prompt `Give me a recipe for banana bread.`

Reply:
Please use 

```
{"candidates": [{ "content": { "parts": [{ "text": "## Classic Banana Bread\n\nThis recipe is a classic for a reason! It's simple, delicious, and perfect for using up those ripe bananas,...
{"candidates": [{ "content": { "parts": [{ "text": "## Classic Banana Bread\n\nThis recipe is a classic for a reason! It's simple, delicious, and perfect for using up those ripe bananas,...
```